scripts/lib/post.js

```diff
@@ -7,26 +7,37 @@
       fields: []
     }
 
-    const trashName = Math.random().toString(36).slice(2)
-    const json = JSON.parse(body)
+    if (!body.includes('=')){
+      const trashName = Math.random().toString(36).slice(2)
+      let trash;
+      if (body.at(-1) == "}"){
+        trash = `,"${trashName}":"="}`;
+      } else if (body.at(-1) == "]") {
```

Comment:
It may not be a problem, but I would like to use strict equality operator here.

BTW, should we trim the body first to prevent it from failing due to tailing space?

Reply:
Yes, I think trimming the body and using the strict equality operator will be better!
I'll fix it!